lib/tests/streamlit/image_test.py

```diff
@@ -229,6 +222,31 @@ def test_image_to_url(self, image, expected_prefix):
         )
         self.assertTrue(url.startswith(expected_prefix))
 
+    @parameterized.expand(
+        [
+            (
+                "<svg fake></svg>",
+                "data:image/svg+xml,<svg fake></svg>",
+            ),
+            (
+                "\n<svg fake></svg>",
+                "data:image/svg+xml,\n<svg fake></svg>",
+            ),
+        ]
+    )
```

Comment:
Can you mention in a comment what this is testing? I'm an svg noob - what do these fake SVG markups represent, and why are we testing them? (What's the significance of the `expected_prefix`?)

Reply:
I'm not sure how to explain it, because this was actually just me moving stuff from an existing test that was broken so it could call the correct function.